<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assignment1Class</class>
 <widget class="QDialog" name="Assignment1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Assignment1</string>
  </property>
  <widget class="QLabel" name="mainMenuTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>SJU_Database_Assignment_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuBtn1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1. File Read (파일 읽기)</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuBtn2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2. File Print (파일 내용 출력)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuBtn3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3. File Update (파일 내용 변경)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuBtn4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4. File Delete (파일 내용 삭제)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuBtn5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>5. File Insert (파일 내용 추가)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuBtn6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>6. Exit (프로그램 종료)</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assignment1.qrc"/>
 </resources>
 <connections/>
</ui>
